### PR TITLE
Fix PositionsControllerTest for good

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix PositionsControllerTest for good
| Type?         | bug fix
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | ~
| How to test?  | ~

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12574)

### CODE REVIEW

File: classes/exception/PrestaShopException.php [hunk 1]
```diff
@@ -96,7 +96,11 @@ public function displayMessage()
         }
         // Log the error in the disk
         $this->logError();
-        exit;
+        if (ToolsCore::isPHPCLI()) {
+            exit(1);
+        } else {
```
Comment (PierreRambaud) [inline classes/exception/PrestaShopException.php]: `exit((int) ToolsCore::isPHPCLI());`

Comment (jolelievre) [reply classes/exception/PrestaShopException.php]: nice!

File: tests-legacy/resources/modules/ps_banner/ps_banner.php [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+                        'type' => 'file_lang',
+                        'label' => $this->trans('Banner image', array(), 'Modules.Banner.Admin'),
+                        'name' => 'BANNER_IMG',
+                        'desc' => $this->trans('Upload an image for your top banner. The recommended dimensions are 1110 x 214px if you are using the default theme.', array(), 'Modules.Banner.Admin'),
+                        'lang' => true,
+                    ),
+                    array(
+                        'type' => 'text',
+                        'lang' => true,
+                        'label' => $this->trans('Banner Link', array(), 'Modules.Banner.Admin'),
```
Comment (LouiseBonnard) [inline tests-legacy/resources/modules/ps_banner/ps_banner.php:175]: Please remove the capital letter: _Banner link_, thanks!

Comment (PierreRambaud) [reply tests-legacy/resources/modules/ps_banner/ps_banner.php:175]: These are tests, no need Wording check, sorry for the inconvenience

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks  @jolelievre

**Pull request merged**